platform/fetch-node/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@affinidi/platform-fetch-node",
+  "version": "1.0.0",
+  "description": "SDK core monorepo for affinity DID solution",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "scripts": {
+    "build": "npm run tsc",
+    "buildDocs": "typedoc --out docs src",
+    "start": "npm run build && node dist/index.js",
+    "tsc": "tsc -p tsconfig-build.json",
+    "lint": "eslint src/** test/**",
+    "test": "npm run test:unit && npm run test:integration",
+    "test:unit": "cross-env NODE_ENV=test mocha -r ts-node/register test/unit/index.test.ts -p ./tsconfig.json --timeout 55000",
+    "test:integration": "cross-env NODE_ENV=test mocha -r ts-node/register test/integration/index.test.ts -p ./tsconfig.json --timeout 115000",
+    "test:coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary mocha -r ts-node/register test/unit/index.test.ts test/integration/index.test.ts -p ./tsconfig.json --timeout 55000",
+    "test:coverage:codeclimate": "npm run test:coverage && $GITHUB_WORKSPACE/tmp/cc-test-reporter format-coverage -t lcov -o $GITHUB_WORKSPACE/tmp/codeclimate.core.json coverage/lcov.info --add-prefix $(pwd | rev | cut -d'/' -f-2 | rev)",
+    "checkLicenses": "license-checker --production --failOn 'AGPL 3.0;LGPL 2.1' --summary"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/affinityproject/affinidi-core-sdk.git",
+    "directory": "sdk/core"
+  },
+  "keywords": [
+    "sdk",
+    "affinity",
+    "network",
+    "did"
+  ],
+  "author": "Denis Popov <denis@slatestudio.com>",
+  "contributors": [
+    "Roman Brazhnyk <roman@affinity-project.org>"
+  ],
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/affinityproject/affinidi-core-sdk/issues"
+  },
+  "homepage": "https://github.com/affinityproject/affinidi-core-sdk#readme",
+  "dependencies": {
+    "@affinidi/platform-fetch": "^1.0.0",
```

Comment:
This should be a peer dependency, but lerna does not support peer dependencies.
Hopefully we can fix this once we switch to npm workspaces.